mapping/requirements_officejs.json

```diff
@@ -7210,6 +7265,17 @@
 								}
 							}],
 							"availability": "GA"
+						},
+												{
+							"name": "Mailbox",
```

Comment:
Support for 1.14 was added for Mac. Should this have been applied to OWA instead?